app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>Recipe: <%= @recipe.name %></p>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+<% @recipe_ingredients.each do |ri| %>
+<p>Ingredient: <%= ri.ingredient.name %></p>
+<% end %>
+<p>Total Cost: <%= @recipe_ingredients.sum do |ri| %> 
```

Comment:
Again this is using Ruby to do the job of ActiveRecord; but this time we're doing Ruby in the view, which since this bit of Ruby is doing data logic, it should be in the Model.